boards/arm/mps4/mps4_corstone315_fvp_ns.yaml

```diff
@@ -1,15 +0,0 @@
-# Copyright 2025 Arm Limited and/or its affiliates <open-source-office@arm.com>
-# SPDX-License-Identifier: Apache-2.0
-
-identifier: mps4/corstone315/fvp/ns
```

Comment:
you are keeping a board target which is wholly untested in zephyr, this is a no, either remove the whole board target and config, or keep the twister file and allow the board target to be tested in CI